text/0000-cargo-post-build-scripts.md

```diff
@@ -0,0 +1,89 @@
+- Feature Name: post_build
+- Start Date: 2016-10-26
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This will add the ability for an extra build script to be run *after* the build completes.
+
+This is a similar concept current build script concept but rather than processing files beforehand,
+this script would perform tasks _after_ the crate has completed compilation.
+
+# Motivation
+[motivation]: #motivation
+
+This feature will give Rust programmers more flexibility when compiling their crates.
+
+Pre-build scripts as we have today are very useful for performing tasks before the
+crate is compiled, such as for compiling C/C++ sources or generating bindings. It
+is not currently possible to do anything _after_ Cargo finishes.
+
+## Use cases
+
+On most platforms, once linking is done there is no longer any work to do. This
```

Comment:
Another common use case for embedded systems is to display the binary size information (e.g. `arm-none-eabi-size the_binary`) after each build.